src/index.js

```diff
@@ -0,0 +1,116 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+const inputTodo = document.querySelector('.add-todo');
+const addTodoBtn = document.querySelector('.submit');
+
+class TodoTask {
+  constructor(desp, arr, done = false) {
```

Comment:
- [ ] It'd be great if we can add arguments that other people that are looking at your code for the first time can understand what is about. Please change 'desp' argument for another more readable.